<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsSC</class>
 <widget class="QDialog" name="SettingsSC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Углы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="angl_cb">
       <item>
        <property name="text">
         <string>Градусы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Радианы</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Линейные коорд.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lin_cb">
       <item>
        <property name="text">
         <string>Метры</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Километры</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ok_pb">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
